README.md

```diff
@@ -159,8 +159,8 @@ the repositories.
    - [pipenv][pipenvdocs] (`pipenv` package on Debian)
 2. **Execute Install Script**: `./scripts/setup_engine.sh`
    ([`scripts/setup_engine.sh`](scripts/setup_engine.sh))
-   1. Clones cc.engine and related respositories
-      - Checks out specified branch (`ARG1`, defaults to `master`)
+   1. Clones cc.engine and related repositories
+      - Checks out specific branch (`ARG1`, defaults to `master`)
```

Comment:
"specified" was actually correct here as we can specify the branch via CLI args.

Reply:
Thanks for the approval. 
I got you.